plugins/modules/zos_lineinfile.py

```diff
@@ -380,6 +387,7 @@ def main():
         backup_name=dict(arg_type="data_set_or_path", required=False, default=None),
         firstmatch=dict(arg_type="bool", required=False, default=False),
         backrefs=dict(arg_type="bool", dependencies=['regexp'], required=False, default=False),
+        tmphlq=dict(type='qualifier_or_empty', required=False, default=None),
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary. As I look more closely, I'm seeing them intermixed. I'll try to put comment notes where I see tmphlq as a dictionary.